lib/errors.js

```diff
@@ -26,39 +26,20 @@ class FederatedError extends Error {
 // allows to copy the `path` & `locations` properties
 // from the already serialized error
 function toGraphQLError (err) {
-  return Object.create(GraphQLError, {
-    name: {
-      value: err.name
-    },
-    message: {
-      value: err.message,
-      enumerable: true,
-      writable: true
-    },
-    locations: {
-      value: err.locations || undefined,
-      enumerable: true
-    },
-    path: {
-      value: err.path || undefined,
-      enumerable: true
-    },
-    nodes: {
-      value: err.nodes || undefined
-    },
-    source: {
-      value: err.source || undefined
-    },
-    positions: {
-      value: err.positions || undefined
-    },
-    originalError: {
-      value: err.originalError || undefined
-    },
-    extensions: {
-      value: err.extensions || undefined
-    }
-  })
```

Comment:
Why this change?

Reply:
I got this error `error.toJSON is not a function`

That's because they changed the `formatError` impl to simply do `return error.toJSON();`

I thought it was safer to move to `new GraphQLError`

Should I go back to `Object.create`?

Doing `Object.create(GraphQLError.prototype,` would also solve the issue